### ISSUE TITLE
MySQL starter commands 

### ISSUE THREAD

**Issue by happycoder0011** [Crio-Bytes/Database#12]
**MicroByte Title:** Basic command in MySQL

**Domain:** DBMS

**Scope of learning for learners:** The beginners who want to get started with MySQL. The byte will include all the basic concepts and commands to create a table, alter table, view table, Primary key and many more things.

**Activities:** Make a student database, create student table, course table and learn update, alter commands.

**Comment by happycoder0011** [comment]
Can i start working on it?

**Comment by archithdwij** [comment]
Good topic! seems beginner friendly. Go ahead with your MicroByte and make sure you maintain the folder structure. 
In the issue, list out the scope and activities in bullet-ins for more readability.
All the best :+1: 
Cheers!

**Comment by YourAKShaw** [comment]
@happycoder0011 We already have a MicroByte which is based on MySQL. 

[MySQL Table with constraints](https://github.com/Crio-Bytes/Database/tree/main/MySQL%20%20Table%20with%20constraints)

Kindly check it out and try to create something new, or try to come up with a new approach with the issue at hand.

**Comment by happycoder0011** [comment]
> @happycoder0011 We already have a MicroByte which is based on MySQL.
> 
> [MySQL Table with constraints](https://github.com/Crio-Bytes/Database/tree/main/MySQL%20%20Table%20with%20constraints)
> 
> Kindly check it out and try to create something new, or try to come up with a new approach with the issue at hand.

yes, I saw that one. I would be majorly focusing on altering, Updating and queries using order by, Group by , having clause.and like.
